grails-app/jobs/mama/ng/scheduler/MessageJob.groovy

```diff
@@ -26,7 +28,13 @@ class MessageJob {
         messages.each { Message message ->
             def schedule = message.schedule
 
-            def success = httpRequestService.postText(schedule.endpoint)
+            def body = [
+                "schedule-id": schedule.id,
+                "message-id": message.id,
+                "send-counter": schedule.sendCounter
```

Comment:
What if the `sendCounter` has changed on the schedule since the Message was created?